rplugin/python3/deoplete/sources/LanguageClientSource.py

```diff
@@ -77,6 +77,7 @@ def gather_candidates(self, context):
                 del self.__results[contextid]
                 if isinstance(items, dict):
                     items = items["items"]
+                items = sorted(items, key=lambda x: x.get("sortText"))
```

Comment:
Just wondering, is it possible some of the items "sortText" is None? In such case, the sort will throw exception since None cannot be compared to str.